docs/release_notes/v1.7.0.md

```diff
@@ -0,0 +1,382 @@
+- **FIXED** AWS SNS/SQS pub/sub: bugfixes - visibility timeout reset; honors disable entity management setting in DLQ  [1445](https://github.com/dapr/components-contrib/issues/1445)
+- **FIXED** Azure CosmosDB state store: Fixes an issue with stored procedure creation when Azure AD credentials are used [1603](https://github.com/dapr/components-contrib/issues/1603)
+- **FIXED** Azure CosmosDB state store: transaction API to respect order of operations [1209](https://github.com/dapr/components-contrib/issues/1209)
+- **FIXED** Azure EventHubs pub/sub: Support Azure AD credentials for checkpointing storage [1566](https://github.com/dapr/components-contrib/issues/1566)
+- **FIXED** Google Cloud pub/sub: Returns No Ack (nack) when there is an error [1561](https://github.com/dapr/components-contrib/issues/1561)
+- **FIXED** mDNS name resolution: sync first browse [1591](https://github.com/dapr/components-contrib/pull/1591)
+- **FIXED** MongoDB state store: Actor reminders broken since 1.5.0 due to data serialization problem (see also hotfix release **1.5.2** and **1.6.1**) [1355](https://github.com/dapr/components-contrib/issues/1355)
+- **FIXED** MongoDB state store: Fixes an issue preventing MongoDB from querying non-string values [1583](https://github.com/dapr/components-contrib/issues/1583)
+- **FIXED** MongoDB state store: Removes MongoDB bson artifacts from JSON value output [1582](https://github.com/dapr/components-contrib/issues/1582)
+- **FIXED** MySQL binding: add `TEXT` column type special case handling [1466](https://github.com/dapr/components-contrib/pull/1466)
+- **FIXED** MySQL binding: adds special handling for `JSON_EXTRACT` and `LONGTEXT` column types [1486](https://github.com/dapr/components-contrib/pull/1486)
+- **FIXED** MySQL state store: transaction API to respect order of operations. [1521](https://github.com/dapr/components-contrib/issues/1521)
+- **FIXED** PostgreSQL state store: transaction API to respect order of operations. [1522](https://github.com/dapr/components-contrib/issues/1522)
+- **FIXED** rabbitmq pub/sub: Fixes a reconnect issue [1471](https://github.com/dapr/components-contrib/pull/1471)
+- **FIXED** Redis state store: Fixes an issue preventing querying of numeric values [1594](https://github.com/dapr/components-contrib/issues/1594)
+- **FIXED** Setting a value using ETag for the Postgres state store [1497](https://github.com/dapr/components-contrib/issues/1497)
+- **FIXED** SQL Server state store: Ensures transaction API executes all transactions sequentially. [1537](https://github.com/dapr/components-contrib/issues/1537)
+- **FIXED** SQL Server state store: transaction API to respect order of operations. [1505](https://github.com/dapr/components-contrib/issues/1505)
+
+### Dashboard
+- **ADDED** Several Dashboard improvements [190](https://github.com/dapr/dashboard/pull/190)
+
+### .NET SDK
```

Comment:
Needs .NET 6.

**ADDED** Build/test with .NET 6 to CI matrix [794](https://github.com/dapr/dotnet-sdk/issues/794)